apps/showcase/src/components/showcase/configuration/configuration-pres.config.ts

```diff
@@ -40,14 +40,14 @@ export interface ConfigurationPresConfig extends Configuration {
   shouldProposeRoundTrip: boolean;
 }
 
-export const CONFIGURATION_PRES_DEFAULT_CONFIG: ConfigurationPresConfig = {
+export const CONFIGURATION_PRES_DEFAULT_CONFIG = {
   inXDays: 7,
   destinations: [
     { cityName: 'London', available: true },
     { cityName: 'Paris', available: true },
     { cityName: 'New-York', available: false }
   ],
   shouldProposeRoundTrip: false
-};
```

Comment:
We should change the generators accordingly and provide a migration script for that no ?

Reply:
Won't migration script be dangerous?

Reply:
This change is a bit dangerous because it can impact the extractor code as well.
Let consider it in different PR is we want this change